html/php-pre-challenge1/index.php

```diff
@@ -1,5 +1,12 @@
 <?php
 for ($i = 1; $i <= 100; $i++) {
-    //ここからコードを書く
-    echo $i;
+    if ( $i % 3 == 0 &&  $i %5 == 0){
+        echo "3の倍数であり、5の倍数<br />";
+    }elseif( $i %5 == 0){
+        echo "5の倍数<br />";
+    }elseif($i %3 == 0){
+        echo "3の倍数<br />";
+    }else{
+        echo $i."<br />";        
```

Comment:
```suggestion
        echo $i . "<br />";
```
ここも、末尾に余計な半角スペースは不要ですね。